src/wallet/salvage.cpp

```diff
@@ -18,6 +18,13 @@ typedef std::pair<std::vector<unsigned char>, std::vector<unsigned char> > KeyVa
 
 bool RecoverDatabaseFile(const fs::path& file_path, bilingual_str& error, std::vector<bilingual_str>& warnings)
 {
+    DatabaseOptions options;
+    DatabaseStatus status;
+    options.require_existing = true;
+    options.verify = false;
+    std::unique_ptr<WalletDatabase> database = MakeDatabase(file_path, options, status, error);
```

Comment:
83a8f51da5bb86c39f030e027100f0eb5f1f52ad, nit:
```suggestion
    auto database = MakeDatabase(file_path, options, status, error);
```